src/less/techne-patterns/_avatars.less

```diff
@@ -212,7 +200,7 @@
 // <div class="row">
 //    <div class="col-xs-12 col-sm-5 col-md-4">
 //      <div class="thumbnail-shape center-block">
-//        <div class="thumbnail__img thumbnail-shape--circle thumbnail--avatar"></div>
+//        <div class="thumbnail__img thumbnail-shape--circle" style="background-image: url('/images/cat-avatar.jpg')"></div>
```

Comment:
@saadmhybris this is the part of task description: * Change the sample background images from CSS classes to inline style: background-image. Because in YaaS usually the background image coming from media service, therefore we can't have it within the LESS files and we could wait quite long until it will be delivered to our app. So in AngularJS this syntax is used:ng-style="{'background-image':'url('+project.imageUrl+')'}" . So we wanted to be as much precise as possible.